packages/lit-dev-content/site/blog/2023-10-10-lit-3.0.md

```diff
@@ -0,0 +1,262 @@
+---
+tags: blogPosts
+layout: blog-post.html
+title: "Lit Launch Day: Lit 3.0, Labs graduations, a compiler and more!"
+summary: "We're launching the next major version of Lit"
+date: 2023-10-10
+author:
+  - lit-team
+---
+
+# Lit Launch Day: Lit 3.0, Labs graduations, a compiler and more!
+
+It's launch day for the Lit team, and we have a bunch of exciting releases to share with the Lit and web components communities!
+
+After several months of development, the Lit team is happy to announce the final release of Lit 3.0 - our fist major version since Lit 2.0 in early 2021 - the first graduating class of Lit Labs packages: `@lit/react`, `@lit/task`, and `@lit/context`, and two bonus releases `@lit-labs/compiler` and `@lit-labs/preact-signals`.
```

Comment:
Replaced first hyphen with em-dash, then a comma.
```suggestion
After several months of development, the Lit team is happy to announce the final release of Lit 3.0 – our fist major version since Lit 2.0 in early 2021, the first graduating class of Lit Labs packages `@lit/react`, `@lit/task`, and `@lit/context`, and two bonus releases `@lit-labs/compiler` and `@lit-labs/preact-signals`.
```